docs/contributing/devguide.md

```diff
@@ -46,6 +52,104 @@ sidebar_position: 3
     - Approvals generally need to come from core members of the [Arrow teams](https://github.com/orgs/Arrow-air/teams).
 5. Merge!
 
+## :zap: VS Code Extensions
+
+These extensions are **recommended**:
+
+```bash
+# In VS Code Quick Open (Ctrl+P)
+ext install \
+bierner.github-markdown-preview \
+rust-lang.rust-analyzer \
+ms-python.python \
+docsmsft.docs-yaml \
+njpwerner.autodocstring \
+stkb.rewrap \
+eamodio.gitlens \
+ms-vscode-remote.vscode-remote-extensionpack \
+tamasfe.even-better-toml \
+```
+
+- [GitHub Markdown Preview](https://marketplace.visualstudio.com/items?itemName=bierner.github-markdown-preview)
+    - Includes `mermaid` (uml) rendering and emoji rendering in markdown preview pane
+- [rust-analyzer](https://marketplace.visualstudio.com/items?itemName=rust-lang.rust-analyzer&ssr=false#overview)
+    - Linting, "go to" support, types and docs on hover, etc.
+- [Python](https://marketplace.visualstudio.com/items?itemName=ms-python.python)
+    - Syntax highlighting, linting, intellisense
+- [docs-yaml](https://marketplace.visualstudio.com/items?itemName=docsmsft.docs-yaml&ssr=false)
+    - Intellisense, validation for YAML files
+- [Even Better
+  TOML](https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml)
+    - TOML files are used for Rust configuration
+    - Validation, syntax highlighting
+- [autoDocstring](https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring)
+    - Python docstring generator
+- [Rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap)
+    - Wrap code comments to fit within n characters
+    - Need to enable this in Preferences->Settings->type "Rewrap"
+    - 80 characters is the recommended column
+- [GitLens](https://marketplace.visualstudio.com/items?itemName=eamodio.gitlens)
+    - Hover over lines to see commit history, last author
+- [Remote Development](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack)
+    - For working over SSH
+
+## :construction: Git Guidelines
+
+### :deciduous_tree: Branches
+
+The branch name should be copied from a [Arrow
+DeWork](https://app.dework.xyz/arrow-air) task.
+
+![DeWork - Suggested Git Branch Name](./assets/dework_branch.png)
```

Comment:
Is this image available?

Reply:
Should be, I can see it generated in the preview

Reply:
It's recommended to put static assets under the static folder.
https://docusaurus.io/docs/static-assets

Otherwise, we might encounter this: https://github.com/facebook/docusaurus/issues/2956

Reply:
Fixed here: https://github.com/Arrow-air/website/pull/41/commits/63a761dcc72fce279150e75d5c3aa8f6da5ed9fe